src/domain/entities/risk-assessment/RiskAssessmentGrading.ts

```diff
@@ -90,18 +92,15 @@ interface RiskAssessmentGradingAttrs extends Ref {
     capacity: CapacityOptions;
     reputationalRisk: WeightedOptions;
     severity: WeightedOptions;
-    // capability: WeightedOptions;
-    grade?: Grade;
+    grade: Maybe<Grade>;
 }
 
 export class RiskAssessmentGrading extends Struct<RiskAssessmentGradingAttrs>() {
     private constructor(attrs: RiskAssessmentGradingAttrs) {
         super(attrs);
     }
 
-    public static createAndCalculateGrade(
-        attrs: RiskAssessmentGradingAttrs
-    ): RiskAssessmentGrading {
+    public static create(attrs: RiskAssessmentGradingAttrs): RiskAssessmentGrading {
         const riskAssessmentGrading = new RiskAssessmentGrading(attrs);
         return riskAssessmentGrading._update({ grade: riskAssessmentGrading.calculateGrade() });
```

Comment:
About that, this kind of calculated properties usually make sense as a `get grade() { return ...}` and remove from Attrs. We can discuss this in the meeting.